packages/grid/x-data-grid/src/components/toolbar/GridToolbarQuickFilter.tsx

```diff
@@ -182,4 +182,11 @@ GridToolbarQuickFilter.propTypes = {
   quickFilterParser: PropTypes.func,
 } as any;
 
+/**
+ * Demos:
+ * - [Filtering - overview](https://mui.com/x/react-data-grid/filtering/)
```

Comment:
Will direct access to the quick filtering page be more meaningful?
```suggestion
 * - [Filtering - quick filter](https://mui.com/x/react-data-grid/filtering/quick-filter/)
```

Reply:
Nice catch ... IMO there is no harm in adding this as an additional ressource, so I just did an additional line linking that detailed demo page 💯

Reply:
That's even better 👍